src/runtime/CMakeLists.txt

```diff
@@ -30,5 +30,5 @@ add_library(dsn_runtime STATIC
         tracer.cpp
         zlocks.cpp
         )
-target_link_libraries(dsn_runtime dsn_utils)
+target_link_libraries(dsn_runtime dsn_utils sasl2 gssapi_krb5 krb5)
```

Comment:
So don't forget to update docs

Reply:
Which docs do you mean?

Reply:
http://pegasus.apache.org/overview/compilation & https://github.com/XiaoMi/pegasus/blob/master/docs/installation.md